dojo/templates/dojo/findings_list_snippet.html

```diff
@@ -91,6 +91,9 @@ <h3 class="has-filters">
                                 <option value="Critical">Critical</option>
                             </select><br/><br/>
 
+                            <label><b>Planned Remediation Date</b><br/>
+                                <input id="id_bulk_planned_remediation_date" name="planned_remediation_date" type="date" alt="Select the date the Finding will be remediated"/></label><br/>
```

Comment:
The placeholder **TT.MM.JJJJ** is bold and black and doesn't look very nice. Can we have this in a normal font with a grey color?

Reply:
Looks like a default Django forms thing. I'll dig into it.